sdk/identity/Azure.Identity/README.md

```diff
@@ -263,6 +263,12 @@ Not all credentials require this configuration. Credentials which authenticate t
 |`AZURE_USERNAME`|a username (usually an email address)
 |`AZURE_PASSWORD`|that user's password
 
+#### Managed identity
+
+|Variable name|Value
+|-|-
+|`AZURE_CLIENT_ID`|Used as the default value for `ManagedIdentityClientId` in `DefaultAzureCredentialOptions`
```

Comment:
```suggestion
|`AZURE_CLIENT_ID`|ID of a Microsoft Entra application. Used as the default value for `ManagedIdentityClientId` in `DefaultAzureCredentialOptions`.
```

Reply:
this would be the client ID for a user-assigned managed identity.

Reply:
Ah, that makes sense. Let's replace my suggestion with something like "The client ID for the user-assigned managed identity."